models/netsuite2/intermediate/int_netsuite2__tran_with_converted_amounts.sql

```diff
@@ -52,16 +52,17 @@ transactions_with_converted_amounts as (
       else false 
         end as is_income_statement,
     case
-      when lower(accounts.type_name) in ('accounts receivable', 'bank', 'deferred expense', 'fixed asset', 'other asset', 'other current asset', 'unbilled receivable') then 'Asset'
+      when lower(accounts.type_name) in ('accounts receivable', 'bank', 'deferred expense', 'fixed asset', 'other asset', 'other current asset', 'unbilled receivable', 'prepaid expense') then 'Asset'
       when lower(accounts.type_name) in ('cost of goods sold', 'expense', 'other expense') then 'Expense'
       when lower(accounts.type_name) in ('income', 'other income') then 'Income'
       when lower(accounts.type_name) in ('accounts payable', 'credit card', 'deferred revenue', 'long term liability', 'other current liability') then 'Liability'
       when lower(accounts.type_name) in ('equity', 'retained earnings', 'net income') then 'Equity'
+      when lower(accounts.type_name) in ('non posting', 'statistical') then 'Other'
```

Comment:
Can you explain what we mean by "Other"? In Accounting I have really only ever been familiar with Asset, Liability, Equity, Revenue, and Expense as these make up the key components of the financial statements. Is the idea that these Other account categories are not brought into the downstream financial statements since they are either non posting or statistical?

Reply:
If the above is true, is there an official category for these or is Other the industry norm?

Reply:
Same question for the NS1 model.

Reply:
I went with "Other" just because of this recommendation from kalor
> Non posting and Statistical are excluded from all financial reports, they are used for other workflows in Netsuite, so it might make sense to add them to a new category called "Non posting" or "Other"

I didn't go with "Non-posting" since there is a non-posting account _type_ (so it felt weird to roll that up with Statistical into a Non-posting category)

if this convolutes things too much though, we could leave it as is (null)

Reply:
Makes sense to me! Would we be able to possibly document this in the DECISIONLOG to explicitly outline why we made this decision.

Reply:
added!